src/main/scala/nl/knaw/dans/narcis/graphql/Command.scala

```diff
@@ -60,6 +60,7 @@ object Command extends App with DebugEnhancedLogging {
     val service = new NarcisGraphqlService(configuration.serverPort, Map(
       "/" -> new NarcisGraphqlRootServlet(configuration.version),
       "/graphql" -> new GraphQLServlet(configuration.profilingThreshold, repository.repository),
+      "/graphiql" -> new GraphiQLServlet("/graphql"),
```

Comment:
Is it common practice to include a client into the server?

Reply:
(1) in most of the projects I've seen (mostly demo projects) on GitHub, a client for experimenting is included
(2) I think it's at least useful to have a dedicated client for GraphQL. It mainly allows you to (a) experiment with the data and (b) browse through the API and documentation quickly.
(3) That is not to say that you cannot have other solutions, but these require extra installation(s). A dedicated client in your gets automatically installed with the server.

Reply:
We can use the GraphiQL standalone application or even Postman. 
It seems nice to have this 'builtin' client, but only if it gives minimal maintenance, so of the shelf libs etc.